aio/.eslintrc.json

```
@@ -0,0 +1,91 @@
+{
+  "root": true,
+  "overrides": [
+    {
+      "files": [
+        "*.ts"
+      ],
+      "parserOptions": {
+        "project": [
+          "tsconfig.json",
+          "tests/e2e/tsconfig.json"
+        ],
+        "createDefaultProgram": true
+      },
+      "extends": [
+        "plugin:@angular-eslint/ng-cli-compat",
+        "plugin:@angular-eslint/ng-cli-compat--formatting-add-on",
+        "plugin:@angular-eslint/template/process-inline-templates"
+      ],
+      "rules": {
+        "@typescript-eslint/ban-types": "error",
+        "@angular-eslint/component-selector": [
+          "error",
+          {
+            "type": "element",
+            "prefix": "aio",
+            "style": "kebab-case"
+          }
+        ],
+        "@angular-eslint/directive-selector": [
+          "error",
+          {
+            "type": "attribute",
+            "prefix": "aio",
+            "style": "camelCase"
+          }
+        ],
+        "dot-notation": "error",
+        "indent": "off",
+        "@typescript-eslint/member-delimiter-style": ["error", {
+          "singleline": {
+            "delimiter": "comma",
+            "requireLast": false
+          }
+        }],
+        "@typescript-eslint/member-ordering": "off",
+        "@typescript-eslint/naming-convention": "off",
+        "no-console": [ "error", {"allow": ["log", "warn", "error"]} ],
```

Super-nit: Let's be consistent with whitespace around array brackets:

```suggestion
        "no-console": ["error", {"allow": ["log", "warn", "error"]}],
```

##
Meta-question: Who lints your lint-config 🤔 😛

---

sorry :bow: 

About the linting on the lint-config, who needs that when we've got such an awesome reviewer!? :heart_eyes: :stuck_out_tongue_winking_eye: 

`gkalpakLint` for the win! :rocket: